scripts/mods/thaumictinkerer.zs

```diff
@@ -1,4 +1,5 @@
 #modloaded thaumictinkerer
+import mods.jei.JEI.removeAndHide as rh;
```

Comment:
Do not use `mods.jei.JEI.removeAndHide` since it prevent debugging of some items.
Use `utils.rh` instead